be/src/vec/data_types/serde/data_type_jsonb_serde.cpp

```diff
@@ -239,6 +239,35 @@ Status DataTypeJsonbSerDe::read_one_cell_from_json(IColumn& column,
                     parser.getWriter().getOutput()->getSize());
     return Status::OK();
 }
+Status DataTypeJsonbSerDe::write_column_to_pb(const IColumn& column, PValues& result, int start,
+                                              int end) const {
+    const auto& string_column = assert_cast<const ColumnString&>(column);
+    result.mutable_string_value()->Reserve(end - start);
+    auto* ptype = result.mutable_type();
+    ptype->set_id(PGenericType::JSONB);
+    for (size_t row_num = start; row_num < end; ++row_num) {
+        const auto& string_ref = string_column.get_data_at(row_num);
+        if (string_ref.size > 0) {
+            std::string json_string =
+                    JsonbToJson::jsonb_to_json_string(string_ref.data, string_ref.size);
+            result.add_string_value(json_string);
```

Comment:
result.add_string_value(JsonbToJson::jsonb_to_json_string(string_ref.data, string_ref.size));

to call std::move